docs/hooks/useUpdate.mdx

```diff
@@ -0,0 +1,65 @@
+---
+name: useUpdate
+menu: Hooks
+---
+
+import { useState } from 'react';
```

Comment:
import React from 'react`

and then in your code do this

```diff
+ React.useEffect(() => {})
- useEffect(() => {})
```

Since React imports the entire package, there is no use for partially importing a module. No DX benefit.